core/src/apps/bitcoin/sign_tx/layout.py

```diff
@@ -58,6 +58,20 @@ async def confirm_output(
     await require_confirm(ctx, text, ButtonRequestType.ConfirmOutput)
 
 
+async def confirm_joint_total(
+    ctx: wire.Context, spending: int, total: int, coin: coininfo.CoinInfo
+) -> None:
+    from trezor.ui.text import Text
+    from apps.common.confirm import require_hold_to_confirm
+
+    text = Text("Joint transaction", ui.ICON_SEND, ui.GREEN)
+    text.normal("You are contributing:")
+    text.bold(format_coin_amount(spending, coin))
+    text.normal("to the total amount:")
```

Comment:
It's the sum of the outputs that the user confirmed, plus the fee. Including the user's change output wouldn't make any sense. It would just confuse the user, because the number would look totally random to him given that his change output is hidden from the confirmation process. An alternative I was considering is below. But what I'd be most interested in is investigating the possibility of hiding the external change outputs as I explained above. It's up for discussion. I copied it into the CoinJoin document we were looking at yesterday so that it doesn't get lost.
![image](https://user-images.githubusercontent.com/42678794/86099730-2bcfaa00-bab8-11ea-97f7-ea103d32ce85.png)

Reply:
> Including the user's change output wouldn't make any sense

I'm not saying it makes any sense.

> I copied it into the CoinJoin document we were looking at yesterday so that it doesn't get lost.

Great.